packages/anoncreds/src/services/AnonCredsIssuerServiceOptions.ts

```diff
@@ -17,11 +22,36 @@ export interface CreateCredentialDefinitionOptions {
   issuerId: string
   tag: string
   supportRevocation?: boolean
-
   schemaId: string
   schema: AnonCredsSchema
 }
 
+export interface CreateRevocationRegistryDefinitionOptions {
+  issuerId: string
+  tag: string
+  credentialDefinitionId: string
+  credentialDefinition: AnonCredsCredentialDefinition
+  maximumCredentialNumber: number
+  tailsDirectoryPath: string
+}
+
+export interface CreateRevocationStatusListOptions {
+  issuerId: string
+  issuanceByDefault: boolean
```

Comment:
I think we should remove this property and always use ISSUANCE_BY_DEFAULT. It's not recommended to use ISSUANCE_ON_DEMAND, and e.g. ACA-Py doesn't allow you to create it with ISSUANCE_ON_DEMAND at all.